nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -272,4 +272,19 @@ config PSA_CRYPTO_DRIVER_ECC_TWISTED_EDWARDS_255_OBERON
 	prompt "PSA ECC Curve Ed25519 support - oberon" if !PSA_PROMPTLESS
 	default y if !PSA_DEFAULT_OFF && PSA_WANT_ECC_TWISTED_EDWARDS_255
 
+config PSA_CRYPTO_DRIVER_JPAKE_OBERON
+	bool
+	prompt "PSA J-PAKE support - oberon" if !PSA_PROMPTLESS
+	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_JPAKE
```

Comment:
Add `depends on PSA_CORE_OBERON` for these three.
I don't think this would compile if you enabled `PSA_CORE_BUILTIN`.